src/services/services.ts

```diff
@@ -3705,6 +3732,41 @@ namespace ts {
                 return entries;
             }
 
+            function getJsDocCompletionEntries(fileName: string, position: number): CompletionEntry[] {
+                let sourceFile = getValidSourceFile(fileName);
+                let hasJsDocComment = ts.hasDocComment(sourceFile, position);
+                if (!hasJsDocComment) {
+                    return undefined;
+                }
+
+                // The current position is right after an At sign

```

Comment:
This comment isn't very helpful; maybe describe what it means if we're after an '@' in a JSDoc comment